CHANGELOG.md

```diff
@@ -6,14 +6,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.11.0] - 2022-10-12
+
 There are incompatible ABI changes in this release. The alphaRange member was
 removed from avifImage struct. The chromaDownsampling and avoidLibYUV members
 were added to the avifRGBImage struct. The imageDimensionLimit member was added
 to the avifDecoder struct. avifImageCopy() and avifImageAllocatePlanes()
 signatures changed. It is necessary to recompile your code. Also check the
 return values of avifImageCopy() and avifImageAllocatePlanes().
 
+### Added
+* Add man pages for avifenc and avifdec
+* Add the avifChannelIndex type alias for enum avifChannelIndex
+* Add avifChromaDownsampling enum
+* Add chromaDownsampling field to avifRGBImage struct
+* Add support for AVIF_RGB_FORMAT_RGB_565
+* Add imageDimensionLimit field to avifDecoder struct
+* Add autoTiling field to avifEncoder struct
+* Add AVIF_CHROMA_DOWNSAMPLING_SHARP_YUV value to avifChromaDownsampling enum
+* Add new avifResult codes AVIF_RESULT_CANNOT_CHANGE_SETTING and
+  AVIF_RESULT_INCOMPATIBLE_IMAGE
+* Add new enum constants AVIF_PIXEL_FORMAT_COUNT and AVIF_RGB_FORMAT_COUNT
+* avifdec: Add --dimension-limit, which specifies the image dimension limit
+  (width or height) that should be tolerated
+* avifenc: Add --sharpyuv, which enables "sharp" RGB to YUV420 conversion, which
+  reduces artifacts caused by 420 chroma downsampling. Needs libsharpyuv (part
+  of the libwebp repository) at compile time.
+* avifenc: Add --ignore-exif and --ignore-xmp flags.
+* avifenc: Add --autotiling, which sets --tilerowslog2 and --tilecolslog2
+  automatically.
+* avifenc: Input Exif orientation is converted to irot/imir by default.
+
 ### Changed
+* Fix memory leaks of metadata on avifenc exit
+* Update the handling of 'lsel' and progressive decoding to AVIF spec v1.1.0
+* Treat an absent lsel and layer_id == 0xFFFF equivalently for backward
+  compatibility with earlier drafts of AVIF spec v1.1.0
+* Set libavif's own default value of cfg.rc_end_usage for libaom
+* Fix https://github.com/AOMediaCodec/libavif/issues/953
```

Comment:
The older entries in this CHANGELOG.md file do not link to issues. Maybe remove them or replace them by a sentence ending with ` (#[number])`.

Reply:
One reason I prefer `Set the cqLevelSet boolean correctly (#953)` over `Fix https://github.com/AOMediaCodec/libavif/issues/953` is that if you happen to search for when `cqLevelSet` behavior changed, you can just search for the "cqLevelSet" string in `CHANGELOG.md`. The opposite holds: if you are looking for something in `CHANGELOG.md` and you know it has nothing to do with `cqLevelSet`, it avoids one click.

Reply:
You are right and these "Fix [bug number]" changelog entries are shortcuts I took when I pored over a lot of commits. I think a good solution is to update the changelog as we write pull requests that should be described in the changelog. Another good solution is to shorten the gap between releases so that there are fewer commits to go through. Or give the release engineer more time to write the changelog.